client/src/main/java/org/camunda/bpm/client/topic/TopicSubscriptionBuilder.java

```diff
@@ -96,6 +98,20 @@
    */
   TopicSubscriptionBuilder processDefinitionVersionTag(String processDefinitionVersionTag);
 
+  /**
+   * @param processVariables to search for external tasks that are supposed to be
+   *                         fetched and locked
+   * @return the builder
+   */
+  TopicSubscriptionBuilder processVariablesContain(Map<String, Object> processVariables);
+
+  /**
+   * @param processVariable to search for external tasks that are supposed to be
+   *                        fetched and locked
+   * @return the builder
+   */
+  TopicSubscriptionBuilder processVariablesContain(String name, Object value);
```

Comment:
See my comment above.

Reply:
Please refer above

Reply:
Here the `processVariableEquals` would make sense, right?

Reply:
This method can be invoked (and the filter variables can be set) more than once and as a result, similar to the above one, ANY match will result in a successful filtering. So, I tend to think, we could use the same method (overload) name as above: `processInstanceVariableIn` ?

Reply:
Do we really need this method or would it from a user perspective also be fine to use `processVariablesIn(Collections.singletonMap("myVariable", "myValue")`? What do you think?

Reply:
I am inclined to think that, it would be better to let both options i.e., `processVariableIn(Map<String,Object>)` and `processVariableIn(String,Object)` be available to choose from the user (i.e., developer) experience point of view even though one might be technically suffice. Your thoughts?

Reply:
I'm not convinced. From a user perspective, it's weird that this method has `In` in its name but only works in an `In` manner when calling the method multiple times, which is only obvious when reading the Javadocs but not from just using this method. 

Wouldn't it be better to use the following names:
* `processVariableEquals` => has Javadocs which explain that calling this method multiple times has OR characteristics
* `processVariablesEqualsIn`

Reply:
Sure and Thanks. Need one more advice on a different note, regarding the next/upcoming commit, to squash or not?

Reply:
Hi @tasso94 , I have made all the necessary changes based on the review comments including the test cases. Please review whenever you could next. Cheers